drracket-tool-text-lib/drracket/private/syncheck/traversals.rkt

```diff
@@ -48,76 +48,81 @@
-                     (annotate-basic sexp
-                                     user-namespace user-directory
-                                     phase-to-binders
-                                     phase-to-varrefs
-                                     phase-to-varsets
-                                     phase-to-tops
-                                     binding-inits
-                                     templrefs
-                                     module-lang-requires
-                                     phase-to-requires
-                                     sub-identifier-binding-directives)
-                     (annotate-variables user-namespace
-                                         user-directory
-                                         phase-to-binders
-                                         phase-to-varrefs
-                                         phase-to-varsets
-                                         phase-to-tops
-                                         templrefs
-                                         module-lang-requires
-                                         phase-to-requires
-                                         sub-identifier-binding-directives)
-                     (annotate-contracts sexp
-                                         (hash-ref phase-to-binders 0 (λ () (make-id-set 0)))
-                                         (hash-ref binding-inits 0 (λ () (make-id-set 0)))
-                                         binder+mods-binder)
-                     (when print-extra-info?
-                       (print-extra-info (list (list 'phase-to-binders phase-to-binders)
-                                               (list 'phase-to-varrefs phase-to-varrefs)
-                                               (list 'phase-to-varsets phase-to-varsets)
-                                               (list 'phase-to-tops phase-to-tops)
-                                               (list 'phase-to-requires phase-to-requires)
-                                               (list 'binding-inits binding-inits)
-                                               (list 'templrefs templrefs)
-                                               (list 'module-lang-requires module-lang-requires)
-                                               (list 'requires requires)
-                                               (list 'require-for-syntaxes require-for-syntaxes)
-                                               (list 'require-for-templates require-for-templates)
-                                               (list 'require-for-labels require-for-labels)
-                                               (list 'sub-identifier-binding-directives
-                                                     sub-identifier-binding-directives)))))]
-                  [else
+              (define is-module?
+                (syntax-case sexp (module)
+                  [(module . rest
+                     )
```

Comment:
I'm thinking a close paren on its own line isn't intentional?

Reply:
Definitely not intentional. Looks like a `fmt` bug, cc @sorawee.

Reply:
Ah, so the issue is that `fmt` thinks that this is a normal `module` formatting, so `.` is the module name and `rest` is the module path. The body is missing, so it places the close paren there.

This issue would affect virtually all forms as well. It's not specific to `module`.

Reply:
I think a heuristic that could work well, without requiring formatting rule writers to add more formatting handlers, is to see if a special form has `.` or not. If there is one, then just treat the form as a function application.

Reply:
Opened sorawee/fmt#82 to track this problem.